src/apps/benchmarks/benchmarkH3Api.c

```diff
@@ -34,4 +34,11 @@ BENCHMARK(h3ToGeoBoundary, 10000, {
     H3_EXPORT(h3ToGeoBoundary)(hex, &outBoundary);
 });
 
+H3Index* out = malloc(H3_EXPORT(maxKringSize)(40) * sizeof(H3Index));
```

Comment:
Could you add a new file for this (`benchmarkKRing.c`)? If you're not comfortable editing `CMakeLists.txt` etc I could break this out to a new benchmark in a separate PR.